extension/api.js

```diff
@@ -0,0 +1,80 @@
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "skeleton" }]*/
+/* global ExtensionAPI */
+
+const PREF_HOTFIX_DONE = "extensions.hotfix-intermediate-2018.done";
+const PREF_MASTER_PASSWORD_REQUEST = "extensions.hotfix-intermediate-2018.requestMasterPassword";
+
+this.hotfixIntermediateCert = class extends ExtensionAPI {
+  onStartup() {
+    if (Services.prefs.getBoolPref(PREF_HOTFIX_DONE, false)) {
```

Comment:
Can we also return early if we know that the version has been patched:
- Version ESR 115.22+
- Version ESR 128.9+
- Version 137+

We know that some users somehow keep the add-on around for no good reasons. Making it no-op limits the impact on the future.

Reply:
the ones that would be getting the patch that adds the extensions.signatureCheckpoint may be (at least partially) covered by the new version of this patch using the same `extensions.signatureCheckpoint` from https://phabricator.services.mozilla.com/D242193

do we feel the need to be explicitly checking the app version in addition to that? (e.g. by using Services.vc and Services.appinfo).